biofefi/options/enums.py

```diff
@@ -87,3 +87,8 @@ class PlotOptionKeys(StrEnum):
 
 class ViewExperimentKeys(StrEnum):
     ExperimentName = "view_experiment_name"
+
+
+class ExplainModels(StrEnum):
+    ExplainModels = "explain_models"
```

Comment:
Can this go under the "Feature Importance options" section of `ConfigStateKeys` for now please? I am thinking that that enum can be broken down but not in this PR.